docs/quickstart.md

```diff
@@ -23,49 +23,44 @@ It  copies the following, necessary files to current directory:
 #!/usr/bin/env sh
 
 VERSION=13
-
-# Copy necessary files to current directory
-
-# Check and get the Gateway Jar
-if [[ -f "gateway-ha.jar" ]]; then
-    echo "Found gateway-har.jar file in current directory."
-else
-    echo "Failed to find gateway-ha.jar in current directory. Fetching version $VERSION from Maven Central repository."
-    curl https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/${VERSION}/gateway-ha-${VERSION}-jar-with-dependencies.jar -o ./gateway-ha.jar
-fi
-
-# Check and get the Config.yaml
-if [[ -f "quickstart-config.yaml" ]]; then
-    echo "Found quickstart-config.yaml file in current directory."
-else
-    cp ../docs/quickstart-config.yaml ./quickstart-config.yaml
-fi
-
-# Check and get the postgres.sql
-if [[ -f "gateway-ha-persistence-postgres.sql" ]]; then
-    echo "Found gateway-ha-persistence-postgres.sql file in current directory."
-else
-    cp ../gateway-ha/src/main/resources/gateway-ha-persistence-postgres.sql ./gateway-ha-persistence-postgres.sql
-fi
-
-#Check if DB is running
-if docker ps --format '{{.Names}}' | grep -q '^local-postgres$'; then
-    echo "PostgreSQL database container 'localhost-postgres' is already running. Only starting Trino Gateway."
-else
-    echo "PostgreSQL database container 'localhost-postgres' is not running. Proceeding to initialize and run database server."
-    export PGPASSWORD=mysecretpassword
-    docker run -v "$(pwd)"/gateway-ha-persistence-postgres.sql:/tmp/gateway-ha-persistence-postgres.sql --name local-postgres -p 5432:5432 -e POSTGRES_PASSWORD=$PGPASSWORD -d postgres:latest
-    #Make sure the DB has time to initialize
-    sleep 5
-
-    #Initialize the DB
-    docker exec local-postgres psql -U postgres -h localhost -c 'CREATE DATABASE gateway'
-    docker exec local-postgres psql -U postgres -h localhost -d gateway -f /tmp/gateway-ha-persistence-postgres.sql
-fi
-
-
-#Start Trino Gateway server.
-java -Xmx1g -jar ./gateway-ha.jar ./quickstart-config.yaml
+BASE_URL="https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha"
+POSTGRES_SQL="gateway-ha-persistence-postgres.sql"
+JAR_FILE="gateway-ha-$VERSION-jar-with-dependencies.jar"
+GATEWAY_JAR="gateway-ha.jar"
+CONFIG_YAML="quickstart-config.yaml"
```

Comment:
The script doesnt actually work .. options:

* remove the log levels line in the config file `log.levels-file: ..`
* add a command here that adds an empty log properties file in the right folder
* add a command here that adds and empty log.properties file in the current folder and update config for current folder

Reply:
Personally I think we dont need the log levels in the quickstart .. what do you think @willmostly ?